spec/whois/parsers/responses/whois.denic.de/de/status_registered_spec.rb

```diff
@@ -23,7 +23,7 @@
 
   describe "#disclaimer" do
     it do
-      expect(subject.disclaimer).to eq("The data in this record is provided by DENIC for informational purposes only. DENIC does not guarantee its accuracy and cannot, under any circumstances, be held liable in case the stored information would prove to be wrong, incomplete or not accurate in any sense. All the domain data that is visible in the whois service is protected by law. It is not permitted to use it for any purpose other than technical or administrative requirements associated with the operation of the Internet. It is explicitly forbidden to extract, copy and/or use or re-utilise in any form and by any means (electronically or not) the whole or a quantitatively or qualitatively substantial part of the contents of the whois database without prior and explicit written permission by DENIC. It is prohibited, in particular, to use it for transmission of unsolicited and/or commercial and/or advertising by phone, fax, e-mail or for any similar purposes. By maintaining the connection you assure that you have a legitimate interest in the data and that you will only use it for the stated purposes. You are aware that DENIC maintains the right to initiate legal proceedings against you in the event of any breach of this assurance and to bar you from using its whois service. The DENIC whois service on port 43 never discloses any information concerning the domain holder/administrative contact. Information concerning the domain holder/administrative contact can be obtained through use of our web-based whois service available at the DENIC website: http://www.denic.de/en/domains/whois-service/web-whois.html")
+      expect(subject.disclaimer).to eq("The above data may only be used within the scope of technical or administrative necessities of Internet operation or to remedy legal problems. The use for other purposes, in particular for advertising, is not permitted. The DENIC whois service on port 43 doesn't disclose any information concerning the domain holder, general request and abuse contact. This information can be obtained through use of our web-based whois service available at the DENIC website: http://www.denic.de/en/domains/whois-service/web-whois.html")
```

Comment:
Please do not make changes to the spec directly. These files are auto-generated as explained at the top of the file.
You should edit the corresponding .expected file, and regenerate the specs with

```
rake spec:generate
```

Reply:
fixed with https://github.com/weppos/whois-parser/pull/111/commits/91685120e4a0b1fc66eba4376a366b8fd48f3e11